File ./assets/implot/.git is a directory while file ./assets/implot-cpy/.git is a regular file
diff -crB ./assets/implot/implot.h ./assets/implot-cpy/implot.h
*** ./assets/implot/implot.h	2024-08-14 11:15:10.644518847 -0300
--- ./assets/implot-cpy/implot.h	2024-08-12 00:56:17.719291495 -0300
***************
*** 45,51 ****
  // [SECTION] Obsolete API
  
  #pragma once
! #include "imgui.h"
  
  //-----------------------------------------------------------------------------
  // [SECTION] Macros and Defines
--- 45,51 ----
  // [SECTION] Obsolete API
  
  #pragma once
! #include "../imgui/imgui.h"
  
  //-----------------------------------------------------------------------------
  // [SECTION] Macros and Defines
diff -crB ./assets/implot/implot_internal.h ./assets/implot-cpy/implot_internal.h
*** ./assets/implot/implot_internal.h	2024-08-14 11:15:10.644518847 -0300
--- ./assets/implot-cpy/implot_internal.h	2024-08-12 00:58:00.757395095 -0300
***************
*** 32,38 ****
  #pragma once
  
  #include <time.h>
! #include "imgui_internal.h"
  
  #ifndef IMPLOT_VERSION
  #error Must include implot.h before implot_internal.h
--- 32,38 ----
  #pragma once
  
  #include <time.h>
! #include "../imgui/imgui_internal.h"
  
  #ifndef IMPLOT_VERSION
  #error Must include implot.h before implot_internal.h
